extension/intents/music/music.toml

```diff
@@ -156,3 +156,36 @@ phrase = "What is playing"
 [[music.showTitle.example]]
 phrase = "What's playing?"
 test = true
+
+[music.volume]
+description = "Adjust music volume"
+match = """
+  (increase | turn up) (music | audio | video |) (volume | the volume) [level=levelUp] 
+  (make it louder | louder) [level=levelUp] 
+  (decrease | turn down) (music | audio | video |) (volume | the volume) [level=levelDown] 
+  (make it softer | softer) [level=levelDown] 
```

Comment:
It would be good to do 1-10 as well, and percents, though that could also be a followup to this PR.

Reply:
Sure, I will add those too. 
I have to ask one thing about these phrases, they are not distinguishing audio and video phrases. Hence the commands containing audio, work for video and commands containing video, work for audio too. 
If I separate them, it would make the phrases different from commonly used phrases. Like "increase volume" is commonly used but not "increase video volume" or "increase audio volume".
What would you suggest for that? Keeping them same or which way we can separate those?

Reply:
I think it's fine to let "increase video volume" increase a music player, if that happens to be the active service. Maybe we'd want to distinguish and use that to at least prefer the controlled service at some point (i.e., prefer YouTube if you say "video"), but... that's only a _maybe_. And it's a lot harder and more fragile. So I think we should stick to the simple thing you've done here and treat them the same.